app/views/homepage/show.html.erb

```diff
@@ -19,6 +19,7 @@
         <td><%= link_to 'Show', list %></td>
         <td><%= link_to 'Edit', edit_list_path(list) %></td>
         <td><%= link_to 'Destroy', list, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to 'Unpin from homepage', homepages_list_path(list), method: :delete %></td>
```

Comment:
do my comments appear if I reply all?  Let's see.

On Thu, Jul 7, 2016 at 4:45 PM, Brian Kung <notifications@github.com> wrote:

> In app/views/homepage/show.html.erb
> <https://github.com/briankung/projectcredo/pull/9#discussion_r69993494>:
>
> > @@ -19,6 +19,7 @@
> >          <td><%= link_to 'Show', list %></td>
> >          <td><%= link_to 'Edit', edit_list_path(list) %></td>
> >          <td><%= link_to 'Destroy', list, method: :delete, data: { confirm: 'Are you sure?' } %></td>
> > +        <td><%= link_to 'Unpin from homepage', homepages_list_path(list), method: :delete %></td>
>
> It's temporary, anyway. ¯_(ツ)_/¯
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/9/files/1d8445d89e43ca129466767f8cab3ce202488147#r69993494>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACKsZMZ0TI6Dykinr4PExiCk5dJe7BF2ks5qTXOUgaJpZM4JGlcb>
> .
>